conf/sample.ini

```diff
@@ -584,6 +584,10 @@
 # Default setting for max attempts to sending alert notifications. Default value is 3
 ;max_attempts = 3
 
+# Makes it possible to enforce a minimal interval between evaluations, to reduce load on the backend
+;min_interval_seconds = 1
+
+
```

Comment:
Nit, but could we drop this extra blank line?

Reply:
sure

Reply:
@papagian Thanks!